tests/tck/utils/assertions.py

```diff
@@ -194,4 +194,8 @@ def assert_resultsets_equals(actual, expected):
     actualCtr = Counter(prepare_actual_row(row) for row in actual.result_set)
     expectedCtr = Counter(prepare_expected_row(row) for row in expected)
     # Validate that the constructed Counters are equal
+    print("actual")
```

Comment:
Not sure we want these prints here

Reply:
well...It helps when your test fails since there is no additional output besides the failure indication. Tell me if you want me to remove